src/actions/security-chat.js

```diff
@@ -37,9 +39,31 @@ async function createMessage(newMessage) {
       body: JSON.stringify(newMessage),
       headers: { 'Content-Type': 'application/json' },
     });
-    const result = await response.json(response);
-    console.info('333', result);
-    return result;
+    return await response.json();
+  } catch (error) {
+    return error;
```

Comment:
All the functions in `actions` folders should be action generators and `createMessage` is not, this function is used only once and definitely was extracted in unnecessary decomposition. Please copy its contents and paste it in an action generator.